support/cas-server-support-thymeleaf/src/main/resources/templates/fragments/pmlinks.html

```diff
@@ -28,12 +28,12 @@
                 onclick="document.getElementById('passwordManagementForm').submit();" />
             <br><br>
         </form>
-        <form th:if="${passwordManagementEnabled}" method="post" id="passwordManagementUserForm">
+        <form th:if="${forgotUsernameEnabled}" method="post" id="forgotUsernameForm">
```

Comment:
Is it possible for one to enable forgot-username functionality without password-management? 

Sounds like the result of `passwordManagementEnabled` should be a AND combination of both password-management and forgot username?

Reply:
Good point! Currently, the "forgot username" link is only displayed if the "password management" is enabled so we cannot use the "forgot username" without the "password management".

We can keep that with: `<form th:if="${passwordManagementEnabled && forgotUsernameEnabled}"` instead of this new test: `<form th:if="${forgotUsernameEnabled}"`.

I guess it's more a functional choice than a technical one.

Reply:
From my experience, I have never seen anyone using the "forgot username" feature without the "password management" feature.

I will change the test.

Reply:
It's committed.

And it's fully backward compatible: I backported it to the 7.0.x stream: https://github.com/apereo/cas/pull/5965